index.html

```diff
@@ -1414,8 +1447,43 @@ <h3>Selective Disclosure</h3>
       <section>
         <h3>Predicates</h3>
         <p>
-Predicates allow a <a>holder</a> to provide True/False values to a
-<a>verifier</a> rather than revealing claim values.
+Another technique which may be used to support progressive trust is to use
+predicates as the values of revealed claims. Predicates allow a <a>holder</a> to
+provide True/False values to a <a>verifier</a> rather than revealing claim
+values.
+        </p>
+        <p>
+Predicate proofs may be enabled by <a>verifiable credential</a> <a>issuers</a>
+as claims, e.g., the <a>credentialSubject</a> may include a
+<code>ageOver18</code> <a>property</a> rather than a <code>birthdate</code>
+<a>property</a>. This would allow <a>holders</a> to provide proofs that they are
+over 18 without revealing their birthdates.
+        </p>
+        <p>
+Certain signature types enable predicate proofs by allowing claims from a
+standard <a>verifiable credential</a> to be presented as predicates. E.g., a
+<a href="https://groups.csail.mit.edu/cis/pubs/lysyanskaya/cl02b.pdf">
+Camenisch-Lysyanskaya signed</a> <a>verifiable credential</a> that contains a
+<code>credentialSubject</code> with a <code>birthdate</code> <a>property</a> may
+be included in a <a>verifiable presentation</a> as a derived credential that
+contains an <code>ageOver18</code> <a>property</a>.
+        </p>
+      </section>
+      <section>
+        <h3>Further Techniques</h3>
+        <p>
+The examples provided in this section are intended to illustrate some possible
+mechanisms for supporting progressive trust, not provide an exhaustive or
+comprehensive list of all the ways progressive trust may be supported. Research
+in this area continues with the use of cutting-edge proof techniques such as
+<a href="https://z.cash/technology/zksnarks/">zk-SNARKS</a> and
+<a href="https://crypto.stanford.edu/bulletproofs/">Bulletproofs</a>, as well as
+different signature protocols.
+        </p>
+        <p>
+A draft report by the <a href="https://w3c-ccg.github.io/">CCG</a> on
```

Comment:
`<a href="https://w3c-ccg.github.io/">CCG</a>`
-> `<a href="https://www.w3.org/community/credentials/">Credentials Community Group</a>`

Reply:
addressed in [a2642eb](https://github.com/w3c/vc-imp-guide/pull/14/commits/a2642eb58336bcc4303270b72f9e3ffcc8cd74bc)